test/api/controllers/requestParsers/validators/validations/DateValidationSpec.scala

```diff
@@ -50,6 +50,14 @@ class DateValidationSpec extends UnitSpec {
 
       }
 
+      "when an out of range fromDate is supplied" in {
+        val invalidFromDate = "1899-01-01"
+        val validationResult  = DateValidation.validate(invalidFromDate, isFromDate = true)
+        validationResult.isEmpty shouldBe false
+        validationResult.length shouldBe 1
+        validationResult.head shouldBe FromDateOutOfRangeError
+      }
+
       "when an invalid toDate is supplied" in {
```

Comment:
Would also include an invalid date like 01-01-0010 which will fail the LocalDate.parse validation

Reply:
Michael, for some reason i dont see the above date mentioned by abigail added to test